package.json

```diff
@@ -86,6 +86,7 @@
     "watch": "lerna run watch --no-sort --stream --concurrency 999"
   },
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "scripts/*"
```

Comment:
Added this so that running `yarn` in the root will also install any additional deps for scripts. Think this makes sense.